proposals/0000-reduce-fragmentation.md

```diff
@@ -0,0 +1,837 @@
+- [ ] `HTMLElement.offsetTop` [Read only]. Returns a double, the distance from this element's top border to its offsetParent's top border.
+- [ ] `HTMLElement.offsetWidth` [Read only]. Returns a double containing the width of an element, relative to the layout.
+
+### Methods
+
+Inherits methods from its parent, Element, and implements those from DocumentAndElementEventHandlers, GlobalEventHandlers, and TouchEventHandlers.
+
+- [x] `HTMLElement.blur()`. Removes keyboard focus from the currently focused element.
+- [ ] `HTMLElement.click()`. Sends a mouse click event to the element.
+- [ ] `HTMLElement.focus(options)`. Makes the element the current keyboard focus.
+
+### Events
+
+Listen to these events using `addEventListener()` or by assigning an event listener to the equivalent component prop.
+
+- [ ] `invalid`. Fired when an element does not satisfy its constraints during constraint validation. Also available via the `onInvalid` prop.
+
+Input events
+
+- [ ] `beforeinput`. Fired when the value of an `<input>`, `<select>`, or `<textarea>` element is about to be modified.
+- [ ] `input`. Fired when the value of an `<input>`, `<select>`, or `<textarea>` element has been changed. Also available via the `onInput` property.
+- [ ] `change`. Fired when the value of an `<input>`, `<select>`, or `<textarea>` element has been changed and committed by the user. Unlike the input event, the change event is not necessarily fired for each alteration to an element's value. Also available via the `onChange` property.
+
+Pointer events
+
+- [ ] `gotpointercapture`. Fired when an element captures a pointer using `setPointerCapture()`. Also available via the `onGotPointerCapture` prop.
+- [ ] `lostpointercapture`. Fired when a captured pointer is released. Also available via the `onLostPointerCapture` prop.
+- [ ] `pointercancel`. Fired when a pointer event is canceled. Also available via the onpointercancel property.
+- [ ] `pointerdown`. Fired when a pointer becomes active. Also available via the onpointerdown property.
+- [ ] `pointerenter`. Fired when a pointer is moved into the hit test boundaries of an element or one of its descendants. Also available via the onpointerenter property.
+- [ ] `pointerleave`. Fired when a pointer is moved out of the hit test boundaries of an element. Also available via the onpointerleave property.
+- [ ] `pointermove`. Fired when a pointer changes coordinates. Also available via the onpointermove property.
+- [ ] `pointerout`. Fired when a pointer is moved out of the hit test boundaries of an element (among other reasons). Also available via the `onPointerOut` prop.
+- [ ] `pointerover`. Fired when a pointer is moved into an element's hit test boundaries. Also available via the `onPointerOver` prop.
+- [ ] `pointerup`. Fired when a pointer is no longer active. Also available via the `onPointerUp` prop.
```

Comment:
All these events (not just pointer) are events that must be W3C compatible from the beginning. Nothing that goes through `EventTarget` should be using the non-standard types for events, even if the `on{Event}` prop API keeps using the non-standard types.